modules/core/test/v2/fixtures/coins/recovery.ts

```diff
@@ -0,0 +1,167 @@
+module.exports.recoverBtcSegwitFixtures = function() {
```

Comment:
not quite static json files but neither are the other files in this dir

at least they are in a separate file ;)